wasm/src/programs/manager/deploy.rs

```diff
@@ -307,4 +311,278 @@ impl ProgramManager {
             TransactionNative::from_deployment(owner, deployment, fee).map_err(|err| err.to_string())?,
         ))
     }
+
+    /// Deploy an Aleo program without synthesizing keys and generating certificates.
+    /// Intended for use with Leo Devnode.
```

Comment:
We should make a comment here that the `getOrInitConsensusHeights` should be called beforehand for it to work properly.